### PR TITLE
Use comint-truncate-buffer to keep the output buffer of projectile-ra…

### PR DESCRIPTION
**Pull request by asok**
…ils-server small

Related to #148

### CODE REVIEW

File: projectile-rails.el [hunk 1]
```diff
@@ -1685,7 +1690,12 @@ Killing the buffer will terminate to server's process."
   (add-hook 'compilation-filter-hook 'projectile-rails-server-compilation-filter)
   (add-hook 'kill-buffer-hook 'projectile-rails-server-terminate t t)
   (add-hook 'kill-emacs-hook 'projectile-rails-server-terminate t t)
-  (setq-local compilation-scroll-output t)
+
+  (require 'comint)
```
Comment (purcell) [inline projectile-rails.el:1694]: This should happen at the top level.

File: projectile-rails.el [hunk 2]
```diff
@@ -1685,7 +1690,12 @@ Killing the buffer will terminate to server's process."
...
+  (add-hook 'compilation-filter-hook 'comint-truncate-buffer)
```
Comment (purcell) [inline projectile-rails.el:1695]: This modifies the hook globally, for all compilations, which I think is not what is intended here.

File: projectile-rails.el [hunk 3]
```diff
@@ -287,6 +287,11 @@ When any of the files are found it means that this is a rails app."
   :group 'projectile-rails
   :type 'boolean)
 
+(defcustom projectile-rails-compilation-buffer-maximum-size 500
+  "The maxium size of the compilation buffer for process started by `projectile-rails-server'"
```
Comment (purcell) [inline projectile-rails.el:291]:
```suggestion
  "The maximum size (in lines) of the compilation buffer used by `projectile-rails-server'."
```

**Comment by asok** [comment]
Thanks for the suggestions. They are applied.
I totally missed the fact that I was adding a hook for all comint buffers. So there's this also https://github.com/asok/projectile-rails/blob/master/projectile-rails.el#L1685 but I'd fix it after merging this change.

**Comment by purcell** [comment]
> So there's this also https://github.com/asok/projectile-rails/blob/master/projectile-rails.el#L1688 but I'd fix it after merging this change.

I fixed that myself, so we're all set. 👍

**Pull request merged**